src/lib/components/NavbarComponent.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+    import upcsi from '$lib/icons/upcsi.svg';
+</script>
+<nav>
```

Comment:
Just a nitpick on the spacing here. Though, I don't mind committing this on your behalf. 👌
```suggestion
</script>

<nav>
```